plugins/org.python.pydev.parser/src/org/python/pydev/parser/grammar_fstrings/grammar_fstrings.jjt

```diff
@@ -177,7 +177,7 @@ void f_string_expr(): {Token start, end, bStart;boolean empty=true;SimpleNode bT
 		            }
 		        )?
 		        (type_conversion())?
-		        (format_spec())?
+		        (format_spec())*
```

Comment:
Humm, actually, instead of having multiple `format_spec()` repeated entries, I think that this should be handled inside the `format_spec()` itself (because conceptually, all that text is inside the format_spec() instead of having multiple `format_spec()` entries from the parsing).

i.e.: where it says `(<TEXT>)+` in `format_spec()`, it should probably be `(<TEXT>|<COLON>)+`